compiler/rustc_error_messages/locales/en-US/parse.ftl

```diff
@@ -238,6 +238,7 @@ parse_const_let_mutually_exclusive = `const` and `let` are mutually exclusive
 
 parse_invalid_expression_in_let_else = a `{$operator}` expression cannot be directly assigned in `let...else`
 parse_invalid_curly_in_let_else = right curly brace `{"}"}` before `else` in a `let...else` statement not allowed
+parse_extra_if_in_let_else = Remove the `if` to construct a `let...else` statement
```

Comment:
```suggestion
parse_extra_if_in_let_else = remove the `if` to construct a `let...else` statement
```
Sorry for being this pedantic, but AFAIK the rest of the help messages begin with a lowercase. So maybe uniformity can be preserved here. Thanks.